src/htmlrenderer.cpp

```diff
@@ -482,6 +489,69 @@ void HtmlRenderer::render(std::istream& input,
 				}
 				break;
 			}
+
+			case HtmlTag::VIDEO: {
+				if (inside_video || inside_audio) {
```

Comment:
Since these variables aren't booleans, it's better to explicitly check for equality: `if (inside_video != 0 || inside_audio != 0)`. (I'd change it in the \<img> code as well, but that's up to you.)

Reply:
Isn't changing them to booleans a better solution? They cannot be nested and I've added the implicit-closing code.

Reply:
Ah, that'd be even better!